draft-ietf-httpbis-header-structure.md

```diff
@@ -191,15 +191,15 @@ The ABNF for dictionaries in HTTP/1 headers is:
 sh-dictionary  = dict-member *( OWS "," OWS dict-member )
 dict-member    = member-name "=" member-value
 member-name    = key
-member-value   = sh-item
+member-value   = sh-item | inner-list
```

Comment:
ABNF alteration is `/` (sorry, I probably misled you into that)

Reply:
Big question here is whether we should refactor List and ListOfLists to be just List with this kind of production as well -- but that's a separable issue.

Reply:
:+1: